homeassistant/components/unifiprotect/sensor.py

```diff
@@ -0,0 +1,443 @@
+        icon="mdi:clock",
+        device_class=DEVICE_CLASS_TIMESTAMP,
+        entity_category=EntityCategory.DIAGNOSTIC,
+        state_class=SensorStateClass.MEASUREMENT,
+        ufp_value="up_since",
+    ),
+    ProtectSensorEntityDescription(
+        key=_KEY_BLE,
+        name="Bluetooth Signal Strength",
+        native_unit_of_measurement=SIGNAL_STRENGTH_DECIBELS_MILLIWATT,
+        device_class=DEVICE_CLASS_SIGNAL_STRENGTH,
+        entity_category=EntityCategory.DIAGNOSTIC,
+        state_class=SensorStateClass.MEASUREMENT,
+        ufp_value="bluetooth_connection_state.signal_strength",
+        ufp_required_field="bluetooth_connection_state.signal_strength",
+    ),
+    ProtectSensorEntityDescription(
+        key=_KEY_WIRED,
+        name="Link Speed",
+        native_unit_of_measurement=DATA_RATE_MEGABITS_PER_SECOND,
+        entity_category=EntityCategory.DIAGNOSTIC,
+        state_class=SensorStateClass.MEASUREMENT,
+        ufp_value="wired_connection_state.phy_rate",
+        ufp_required_field="wired_connection_state.phy_rate",
+    ),
+    ProtectSensorEntityDescription(
+        key=_KEY_WIFI,
+        name="WiFi Signal Strength",
+        native_unit_of_measurement=SIGNAL_STRENGTH_DECIBELS_MILLIWATT,
+        device_class=DEVICE_CLASS_SIGNAL_STRENGTH,
+        entity_category=EntityCategory.DIAGNOSTIC,
+        state_class=SensorStateClass.MEASUREMENT,
+        ufp_value="wifi_connection_state.signal_strength",
+        ufp_required_field="wifi_connection_state.signal_strength",
+    ),
+)
+
+CAMERA_SENSORS: tuple[ProtectSensorEntityDescription, ...] = (
+    ProtectSensorEntityDescription(
+        key=_KEY_RX,
+        name="Received Data",
+        native_unit_of_measurement=DATA_BYTES,
+        entity_registry_enabled_default=False,
+        entity_category=EntityCategory.DIAGNOSTIC,
+        state_class=SensorStateClass.TOTAL_INCREASING,
+        ufp_value="stats.rx_bytes",
+    ),
+    ProtectSensorEntityDescription(
+        key=_KEY_TX,
+        name="Transferred Data",
+        native_unit_of_measurement=DATA_BYTES,
+        entity_registry_enabled_default=False,
+        entity_category=EntityCategory.DIAGNOSTIC,
+        state_class=SensorStateClass.TOTAL_INCREASING,
+        ufp_value="stats.tx_bytes",
+    ),
+    ProtectSensorEntityDescription(
+        key=_KEY_OLDEST,
+        name="Oldest Recording",
+        device_class=DEVICE_CLASS_TIMESTAMP,
+        entity_category=EntityCategory.DIAGNOSTIC,
+        state_class=SensorStateClass.MEASUREMENT,
+        ufp_value="stats.video.recording_start",
+    ),
+    ProtectSensorEntityDescription(
+        key=_KEY_USED,
+        name="Storage Used",
+        native_unit_of_measurement=DATA_BYTES,
+        entity_category=EntityCategory.DIAGNOSTIC,
+        state_class=SensorStateClass.MEASUREMENT,
+        ufp_value="stats.storage.used",
+    ),
+    ProtectSensorEntityDescription(
+        key=_KEY_WRITE_RATE,
+        name="Disk Write Rate",
```

Comment:
Isn't this a high frequency updating sensor? If so, this one should be disabled by default.

Reply:
It _can_ update at a high frequency, but it should be relatively static since it is not like the bitrate for the camera is relatively static (only changes when the user changes quality settings).